src/components/Entry.vue

```diff
@@ -62,7 +62,8 @@ export default {
                 this.$store.state.users[0].status = 'socialLogged'
```

Comment:
semicolons (here, 1 line below and in this file above, sorry can't comment on the line above)